cartridges/app_storefront_custom/cartridge/controllers/Telerik.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+/**
+ * @namespace Home
+ */
+
+var server = require("server");
+var cache = require("*/cartridge/scripts/middleware/cache");
+var consentTracking = require("*/cartridge/scripts/middleware/consentTracking");
+var pageMetaData = require("*/cartridge/scripts/middleware/pageMetaData");
+var userLoggedIn = require("*/cartridge/scripts/middleware/userLoggedIn");
+
+/**
+ * Any customization on this endpoint, also requires update for Default-Start endpoint
+ */
+/**
+ * Home-Show : This endpoint is called when a shopper navigates to the home page
+ * @name Base/Home-Show
+ * @function
+ * @memberof Home
+ * @param {middleware} - consentTracking.consent
+ * @param {middleware} - cache.applyDefaultCache
+ * @param {category} - non-sensitive
+ * @param {renders} - isml
+ * @param {serverfunction} - get
+ */
+server.get(
```

Comment:
Is this file relevant to the homework?

Reply:
This was from lecture video

Reply:
Okay, but if something is going to be part of the repo, lets have more meaningful names, that describe better the function of the file.